live-engage.android.ts

```diff
@@ -119,4 +119,25 @@ export class LiveEngage implements CommonLiveEngage {
     public parsePushMessage(data: any): any {
         return new com.liveperson.infra.messaging_ui.uicomponents.PushMessageParser(data);
     }
+
+    public killChat(): void {
+        if (!this.brandId || !this.appId) {
+            return;
+        }
+
+        if (!this.isValidState()) {
+            return;
+        }
+
+        com.liveperson.messaging.MessagingFactory.getInstance().getController().resolveConversation(this.brandId, this.brandId);
+        const LogoutCallback: any = com.liveperson.infra.callbacks.LogoutLivePersonCallBack.extend({
+            onLogoutSucceed: () => { },
+            onLogoutFailed: (err: any) => {
+                console.error(err);
+            }
+        });
+        const initProperties = new com.liveperson.infra.InitLivePersonProperties(this.brandId, this.appId, null);
+        const ui = new com.liveperson.infra.messaging_ui.MessagingUiInitData(initProperties, this.getSDKVersion());
+        com.liveperson.infra.messaging_ui.MessagingUIFactory.getInstance().logout(application.android.context, ui, new LogoutCallback());
```

Comment:
According to the documentation this is the correct way to remove all user data. This also unregisters for push notifications.